### ISSUE TITLE
ARIA attributes not handled properly

### ISSUE THREAD

**Issue by danielnixon** [bitinn/vdom-parser#15]
It looks like ARIA attributes (e.g. `aria-valuenow`) aren't being parsed properly.

This `data-*` special case should probably apply to `aria-*` too:

```
    // special cases for data attribute, we default to properties.attributes.data
    if (name.indexOf('data-') === 0) {
        value = attr.value;
        isAttr = true;
    } else {
        value = attr.value;
    }
```

**Comment by danielnixon** [comment]
See https://github.com/Matt-Esch/virtual-dom/blob/dcb8a14e96a5f78619510071fd39a5df52d381b7/docs/vnode.md#aria-attributes

**Comment by bitinn** [comment]
I will take a look when available, haven't been using `virtual-dom` lately, so PR welcomed.

**Comment by danielnixon** [comment]
PR created. :)

https://github.com/bitinn/vdom-parser/pull/16/files

**Comment by bitinn** [comment]
Thx, tested and released as v1.3.2
